c_binding/include/libstoragemgmt/libstoragemgmt_nfsexport.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 #define LSM_NFS_EXPORT_ANON_UID_GID_NA      -1
 #define LSM_NFS_EXPORT_ANON_UID_GID_ERROR   -2
 
-/**
+/*
```

Comment:
These functions are plugin only function, I don't want to expose the documents of those plugin only functions yet.